configs/containers/specs/jedi-ci.yaml

```diff
@@ -38,7 +38,6 @@
     py-f90nml,
     py-gitpython,
     py-h5py,
-    py-numpy,
```

Comment:
I assume you are removing this because it is implicitly included with py-pandas?

Reply:
correct - unnecessary inclusion and earlier when there was a version specified it was messing with concretize somehow due to the intel variant. I probably could have fixed it, but since it is implicitly included it isn't necessary in the spec.